test/minitest_ext.rb

```diff
@@ -56,12 +56,14 @@ def run_cmd(cmd, split_cmd = true)
       ShopifyCLI::Core::EntryPoint.call(new_cmd, @context)
     end
 
-    def capture_io(&block)
+    def capture_io(filter_colours: false, &block)
       cap = CLI::UI::StdoutRouter::Capture.new(with_frame_inset: true, &block)
       @context.output_captured = true if @context
       cap.run
       @context.output_captured = false if @context
-      [cap.stdout, cap.stderr]
+      [cap.stdout, cap.stderr].map do |s|
+        filter_colours ? s.gsub(/\e\[\d+(;\d+)?m/, "") : s
```

Comment:
We can take this one step farther and generalize this to remove all ANSI escape sequences

```ruby
strip_ansi ? CLI::UI::ANSI.strip_codes(s) : s
```